### PR TITLE
Add Framework jumper guide

### PR DESCRIPTION
**Pull request by TomasVotruba**
### Version 2018-04-01

image

### CODE REVIEW

**Comment by TomasVotruba** [comment]
### Version few hours later

https://projects.invisionapp.com/d/main#/console/13846081/288527845/preview
image

File: source/_data/talks.yml [hunk 1]
```diff
@@ -6,6 +6,7 @@ parameters:
             event_name: "PHPLive 2018"
             event_link: "https://www.phplive.cz/"
             slides_link: "https://docs.google.com/presentation/d/1YFnNZeC7uEn5bpFIoGuEjdYLLkE0o3nFq7HGMlWENY4/edit?usp=sharing"
+            video_link: "https://www.youtube.com/watch?v=QbM1WYOTozI"
```
Comment (carusogabriel) [inline source/_data/talks.yml]: Time to train my Crezh :nerd_face:

File: source/framework-jumper-guide.latte [hunk 2]
```diff
@@ -0,0 +1,50 @@
+---
+layout: default
+title: "How Code Looks in every PHP Framework?"
+id: framework-jumper-guider
+---
+
+{block content}
+    <div class="row">
+        <div class="col-md-12 text-center">
+            <h1>How Code Looks in every PHP Framework?</h1>
```
Comment (carusogabriel) [inline source/framework-jumper-guide.latte]:
```diff
-How Code Looks in every PHP Framework?
+How Does Code Look in every PHP Framework?
```

**Comment by TomasVotruba** [comment]
Mergin as prototype is ready :+1:

**Comment by TomasVotruba** [comment]
We're online :rocket: 

https://www.tomasvotruba.cz/framework-jumper-guide/

Test on your :iphone:

**Pull request merged**